### PR TITLE
object: Export constant key to the expiration attribute

### PR DESCRIPTION
**Pull request by cthulhu-rider**

### CODE REVIEW

**Review by roman-khimov — APPROVED** [review]

File: object/attribute.go [hunk 1]
```diff
@@ -4,6 +4,22 @@ import (
...
+	// attribute never expire.
+	//
+	// Reaction of NeoFS system components to the objects' 'expired' property may
+	// vary. For example, in the basic scenario, expired objects are auto-deleted
+	// from the storage. Detailed behavior can be found in the NeoFS Specification.
+	//
+	// Note that the value determines exactly the last epoch of the object's
+	// relevance: for example, with the value N, the object is relevant in epoch N
+	// and expired in any epoch starting from N+1.
+	AttributeExpirationEpoch = object.SysAttributeExpEpoch
```
Comment (carpawell) [inline object/attribute.go:20]: do we need "key", "system" words here?

**Review by carpawell — APPROVED** [review]

File: object/attribute.go (hunk 1 above)
Comment (cthulhu-rider) [reply object/attribute.go:20]: `key` is missing in existing ones, `system` - may be added, but i don't think user will care much and we'll have non-system expiration attribute

u think?

Comment (carpawell) [reply object/attribute.go:20]: i dont mind keeping it that way

Comment (roman-khimov) [reply object/attribute.go:20]: "key" is kinda obvious (in that it's not a value, but some attribute name/key). "system" as well, we don't have constants for random attributes here, we only have ones that have some meaning to the system.

**Pull request merged**